package.json

```diff
@@ -44,5 +44,8 @@
   "devDependencies": {
     "elm-doc-preview": "^5.0.5",
     "elm-review": "^2.4.4"
+  },
+  "resolutions": {
```

Comment:
We won't need this if we merge #192  first.
But it's also an option to merge this one first and add a commit to #192 removing this.